src/client/src/Domains/FacultyMember/api/api.ts

```diff
@@ -0,0 +1,65 @@
+import API from 'api';
+
+export const departments = [
```

Comment:
I noticed the data from this departments array of objects is also used in the JobForm component. I think we can move this array somewhere outside of the domains folder. Maybe create a file called "sharedData.ts" in src. Then, all components that need to get data about the departments will just get it from that shared file (there is only one source of truth, we don't need to update department's data in multiple places). You will need to refactor the JobForm component too (should be simple).